README.md

```diff
@@ -1,61 +1,2577 @@
+balenalib/intel-nuc-debian-node   latest              4498fe12b99e        38 hours ago        270MB
+```
+As you can see, because when we rebuilt we only specified the repository name
+to `-t` and *not* an actual tag, `latest` was assumed and the `latest` tag now
+points to the newer image.
+
+It's always a really good idea to specifically tag the images you're building
+with a suffix. This stops older built images from losing their repository and
+tag name, which allows you to easily identify older versions of the image.
+
+To reinforce that point, we'll add one final comment line to `index.js`, just
+below the other:
+```
+// Another comment changes the image
+```
+Finally, rebuild the image with a `v0.0.2` tag *and* the `latest` tag:
+```
+$ docker build -t simple-docker-app:v0.0.2 -t simple-docker-app:latest .
+Sending build context to Docker daemon   5.12kB
+Step 1/6 : FROM balenalib/intel-nuc-debian-node:latest
+ ---> 4498fe12b99e
+Step 2/6 : EXPOSE 9854
+ ---> Using cache
+ ---> ff63da014498
+Step 3/6 : WORKDIR /usr/src/app
+ ---> Using cache
+ ---> 568a5e43a29e
+Step 4/6 : COPY * /usr/src/app/
+ ---> ba616c670298
+Step 5/6 : RUN npm install
+ ---> Running in 0201cb715e8a
+npm notice created a lockfile as package-lock.json. You should commit this file.
+npm WARN simple-docker-app@1.0.0 No description
+npm WARN simple-docker-app@1.0.0 No repository field.
+
+added 50 packages from 37 contributors and audited 126 packages in 2.677s
+found 0 vulnerabilities
+
+Removing intermediate container 0201cb715e8a
+ ---> b20eabc15eb0
+Step 6/6 : CMD ["/usr/local/bin/npm", "start"]
+ ---> Running in dceef18cc322
+Removing intermediate container dceef18cc322
+ ---> d7fefc78916d
+Successfully built d7fefc78916d
+Successfully tagged simple-docker-app:v0.0.2
+Successfully tagged simple-docker-app:latest
+```
+And run:
+```
+$ docker images
+REPOSITORY                        TAG                 IMAGE ID            CREATED              SIZE
+simple-docker-app                 latest              d7fefc78916d        22 seconds ago       273MB
+simple-docker-app                 v0.0.2              d7fefc78916d        22 seconds ago       273MB
+<none>                            <none>              4817e03f7def        About a minute ago   273MB
+simple-docker-app                 v0.0.1              25d3625211c2        3 minutes ago        273MB
+balenalib/intel-nuc-debian-node   latest              4498fe12b99e        38 hours ago         270MB
+```
+We now have a new version, `v0.0.2`, which is also the `latest` version. We also
+still have our original version, `v0.0.1`. But note that the version we built
+without a tag has now been stripped of its repository and `latest` tag. Because
+Docker was not given a full `repository:tag` name for the image, and because
+we've moved the `latest` tag on, it's had to remove the name and tag from that
+image, and it is now only know by it's original ID (`4817e03f7def`).
+
```

Comment:
Also worth a mention or a link to `docker tag` command?

Reply:
I actually reference it in the repository section where it gets used, so it should be clearer there.